install_config/master_node_configuration.adoc

```diff
@@ -33,259 +33,698 @@ remember all of the options to specify. To make this easier, the configuration
 files can be created with the `--write-config` option and then used with the
 `--config` option.
 
-[[creating-new-configuration-files]]
+[[master-configuration-files]]
+== Master Configuration Files
+This section reviews parameters mentioned in the *_master-config.yaml_* file.
 
-== Creating New Configuration Files
-For masters, the `openshift start` command accepts options that indicate that it
-should simply write the configuration files that it would have used, then
-terminate. For nodes, a configuration file can be written using the `oadm
-create-node-config` command. Creating new configuration files is useful to get a
-starting point for defining your configuration.
+You can xref:creating-new-configuration-files[create a new node configuration
+file] to see the valid options for your installed version of {product-title}.
 
-The following commands write the relevant launch configuration file(s),
-certificate files, and any other necessary files to the specified
-`--write-config` or `--node-dir` directory.
+[[master-config-admission-control-config]]
+=== Admission Control Configuration
 
-To create configuration files for an all-in-one server (a master and a node on
-the same host) in the specified directory:
+.Admission Control Configuration Parameters
+[cols="3a,6a",options="header"]
+|===
 
-[options="nowrap"]
-----
-$ openshift start --write-config=/openshift.local.config
-----
+| Parameter Name | Description
 
-To create a xref:master-configuration-files[master configuration file] and
-other required files in the specified directory:
+|`*AdmissionConfig*`
+|Contains admission control plug-in configuration.
 
-[options="nowrap"]
-----
-$ openshift start master --write-config=/openshift.local.config/master
-----
+|`*APIServerArguments*`
+|Key value pairs that will be passed directly to the Kube API server that match
+the API servers' command line arguments. These are not migrated, but if you
+reference a value that does not exist the server will not start. These values
+may override other settings in `*KubernetesMasterConfig*`, which may cause
+invalid configurations.
 
-To create a xref:node-configuration-files[node configuration file] and other
-related files in the specified directory:
+|`*ControllerArguments*`
+|Key value pairs that will be passed directly to the Kube controller manager
```

Comment:
@ahardin-rh I just did a search, and sometimes in the docs we use "key-value". I'm not sure which is more accurate, but I'd lean towards that one. We're pretty loose with this though, so I'd say both are acceptable. Developing a standard might be a good idea for the future (brice thinks to himself...).